packages/perseus/src/widgets/radio/radio-component.tsx

```diff
@@ -27,6 +27,7 @@ export type RenderProps = {
     countChoices?: boolean;
```

Comment:
Same note here as radio.class.new, so I'll copy the comment: I've deleted this file in my restructure PR, but I'll happily pull this work over into that PR before landing. (I'll let you land first). I'll send you a message once I've integrated your changes, just so that you can review to verify that I didn't miss anything. :)